aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -803,6 +809,12 @@ Tensor& argmin_out(Tensor& result, const Tensor& self, c10::optional<int64_t> di
       "tensor with no elements because the operation does not have an identity");
   Tensor in;
   if (dim) {
+    if (self.sizes()[dim.value()] == 1) {
+      auto sizes = self.sizes().vec();
+      sizes.erase(sizes.begin() + dim.value());
```

Comment:
What happens if `keepdim=True`?

Reply:
Thanks for pointing that out! I’ll fix it and add a corresponding test.